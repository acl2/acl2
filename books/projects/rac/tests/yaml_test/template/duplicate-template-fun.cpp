// RAC begin

int foo() { return 0; }

template <int x>
int foo() { return 0; }
