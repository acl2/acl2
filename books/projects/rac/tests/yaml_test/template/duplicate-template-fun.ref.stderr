duplicate-template-fun.cpp:5-6 (0-9): Duplicate identifier declaration (could be an enum, a variable template parameter or function) `foo`
 5 | template <int x>
   | ^^^^^^^^^^^^^^^^^
 6 | int foo() { return 0; }
   | ^^^^^^^^^
note: first defined here
 3 | int foo() { return 0; }
   | ^^^^^^^^^
